src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -27,16 +27,46 @@
 
 (defn clear-both [] [:div {:style {:clear "both"}}])
 
-(defn scroll-to [x y] (.scrollTo js/window x y))
-
-(defn scroll-to-top [] (scroll-to 0 0))
-
-(defn scroll-to-center [elem]
-  (let [elem-center-x (+ (.-offsetLeft elem) (/ (.-offsetWidth elem) 2))
-        elem-center-y (+ (.-offsetTop elem) (/ (.-offsetHeight elem) 2))]
-    (scroll-to
-      (- elem-center-x (/ (.-innerWidth js/window) 2))
-      (- elem-center-y (/ (.-innerHeight js/window) 2)))))
+;; Smooth step from https://en.wikipedia.org/wiki/Smoothstep
+(defn- smooth-step [start end point]
+  (let [x (/ (- point start) (- end start))]
+    (* x x (- 3 (* 2 x)))))
+
+(defn- smoother-step [start end point]
+  (let [x (/ (- point start) (- end start))]
+    (* x x x (+ (* x (- (* x 6) 15)) 10))))
+
+(defn- animate [start-time end-time start-x start-y end-x end-y]
+  (let [now (js/Date.now)]
+    (if (> now end-time)
+      (.scrollTo js/window end-x end-y)
+      (let [point (smooth-step start-time end-time (js/Date.now))]
+        (.scrollTo js/window
+          (+ start-x (* point (- end-x start-x)))
+          (+ start-y (* point (- end-y start-y))))
+        (js/setTimeout #(animate start-time end-time start-x start-y end-x end-y) 10)))))
+
+(defn scroll-to
+  ([x y] (.scrollTo js/window x y))
+  ([x y duration]
```

Comment:
I suggest:
```Clojure
(assert (<= duration 400) "Duration too long (> 400ms)")
```
to ensure that the caller cannot accidentally create a bad experience for the user.